scripts/upgrade

```diff
@@ -1,56 +1,139 @@
 #!/bin/bash
```

Comment:
I think, something is missing in the upgrade script:
https://www.mantisbt.org/docs/master/en-US/Admin_Guide/html/admin.install.upgrade.html

You must curl the `/admin/install.php?submit` page to complete the upgrade